pysrc/bytewax/connectors/kafka/operators.py

```diff
@@ -0,0 +1,309 @@
+"""Operators for the kafka source and sink."""
+from dataclasses import dataclass
+from typing import (
+    Dict,
+    Generic,
+    List,
+    Optional,
+    Tuple,
+    cast,
+)
+
+from confluent_kafka import OFFSET_BEGINNING
+from confluent_kafka.error import KafkaError
+
+import bytewax.operators as op
+from bytewax.dataflow import Dataflow
+from bytewax.operators import Stream, operator
+
+from ._types import K2, V2, K, KafkaKey, KafkaValue, V
+from .message import KafkaMessage
+from .serde import SchemaDeserializer, SchemaSerializer
+from .sink import KafkaSink
+from .source import KafkaSource
+
+__all__ = [
+    "deserialize",
+    "deserialize_key",
+    "deserialize_value",
+    "serialize",
+    "serialize_key",
+    "serialize_value",
+    "kafka_in",
+    "kafka_out",
+]
+
+
+@dataclass(frozen=True)
+class KafkaStreams(Generic[K, V, K2, V2]):
```

Comment:
I agree on changing the name of this class, but it is not strictly related to `Serde`, since I use it in the `input` operator too to split messages with any kind of error too.
Maybe `SplitKafkaErrorOut`? Open to better suggestions here